fish-rust/src/builtins/test.rs

```diff
@@ -84,8 +84,7 @@ mod test_expressions {
         // Return true if the number is a tty().
         fn isatty(&self, streams: &mut IoStreams) -> bool {
             fn istty(fd: libc::c_int) -> bool {
-                // Safety: isatty cannot crash.
-                unsafe { libc::isatty(fd) > 0 }
+                crate::nix::isatty(fd)
```

Comment:
I think we should generally use unqualified function names

I've been using the `libc::` qualifier for C functions, to give a hint to folks who don't know the function.
But no qualifier for constants like `SIGHLD` because those are usually used in a `libc::` function call already.

Not sure what's the future. Here I think it's fine to use unqualified `isatty(fd)` always